IMAGE_README.md

```diff
@@ -0,0 +1,123 @@
+- **Supported architectures:** [...]
+
+## What is this coin with a Doge on it ?
+
+Dogecoin is a community-driven cryptocurrency that was inspired by a Shiba Inu meme. The Dogecoin Core software allows anyone to operate a node in the Dogecoin blockchain networks and uses the Scrypt hashing method for Proof of Work. It is adapted from Bitcoin Core and other cryptocurrencies.
+
+To get more information about Dogecoin Core, visit the [source code repository](https://github.com/dogecoin/dogecoin).
+
+## How to use this image
+
+Start your node in a single command, exposing P2P & RPC ports, mapping the data directory:
+```bash
+docker run -d --name dogecoin-node \
+    -v $(pwd)/datadir:/dogecoin/.dogecoin \
+    -p 22555:22555 \
+    -p 22556:22556 \
+    dogecoin/dogecoin:latest
+```
+It will launch the synchronization of the blockchain, which take hours. That's it, your node is up !
+
+To verify if the container is running and to get logs:
+```bash
+docker ps
+docker logs dogecoin-node
+```
+
+Call the JSON-RPC API using the `dogecoin-cli` present inside the container. The help menu will display all existing commands of the API:
+```bash
+docker exec dogecoin-node dogecoin-cli help
+```
+
+## Syntax & Configuration
+
+```bash
+# Syntax
+docker run  [docker-options] dogecoin/dogecoin [executable] [executable-options]
+```
+
++ `docker-options` : Set environment variables, ports, volumes and other docker settings.  
++ `executable` : Choose between `dogecoind`, `dogecoin-cli`, `dogecoin-tx` or `dogecoin-qt`. Default to `dogecoind`.  
++ `executable-options` : Pass options directly to the executable.
+
+There is three ways to configure the Dogecoin node. By using environment variables, by passing arguments to the executable or by providing `dogecoin.conf` in a volume.
+
+**To see all available configurations, see the `-help` menu from each executable:**
+```bash
+docker run dogecoin/dogecoin -help
+```
+
+### Configure with executable arguments
+
+`docker run` arguments are passed directly to the executable.
+```bash
+docker run dogecoin/dogecoin -paytxfee=0.01 -testnet
+```
+
+### Configure with environment variables
+
+All options for each executable can be defined with environment variables. Variables will be converted to executable arguments.
+
+```bash
+docker run -e PAYTXFEE=0.01 -e TESTNET=1 dogecoin/dogecoin
+```
+Environment variables represent executable arguments converted in upper case, and `-` to `_`.  
+For example, `-help-debug` will be `HELP_DEBUG`.
+
+### Configure with a configuration file
+
+`dogecoin.conf` file can be used to configure your node. This file need to be located in the `datadir` of the node.
+
+Create `dogecoin.conf` with the following content:
+```
+testnet=1
+paytxfee=0.01
+```
+
+Put this file in the directory mounted to `/dogecoin/.dogecoin`, in `datadir` for this example :
+```bash
+docker run -v $(pwd)/datadir:/dogecoin/.dogecoin dogecoin/dogecoin
+```
+
+### Docker Compose
+
+Example for `docker-compose.yml` :
```

Comment:
May want to add a volume assignment to this?

Reply:
done